### ISSUE TITLE
None becomes "None"

### ISSUE THREAD

**Issue by jorndoe** [sanand0/xmljson#29]
https://github.com/sanand0/xmljson/blob/93011b14d1536e01283668e377684eef0a2da279/xmljson/__init__.py#L182

This line should be:

```
            if root.text is None:
                return ''
            else:
                return self._fromstring(root.text)
```

Not good:
`<empty/>` → `{"empty": null}` → `<empty>None</empty>`

Better:
`<empty/>` → `{"empty": null}` → `<empty></empty>`

**Comment by dagwieers** [comment]
@jorndoe Can you make a PR ?

PS You don't need any Git skills, you can simply [edit this file](https://github.com/sanand0/xmljson/edit/master/xmljson/__init__.py#L182) on GitHub and make the required change. Then it will make a PR from it.

**Comment by jorndoe** [comment]
@dagwieers 
Hereby done
PR #30

**Comment by sanand0** [comment]
Closed via 824e39b
